@xen-orchestra/web/src/components/backups/configuration/BackupJobSettings.vue

```diff
@@ -3,62 +3,49 @@
     <UiTitle>{{ t('settings') }}</UiTitle>
     <VtsColumns>
       <VtsColumn>
-        <VtsQuickInfoRow :label="t('proxy')" :value="proxy?.name" />
-        <VtsQuickInfoRow :label="t('snapshot-mode')">
+        <UiLabelValue :label="t('proxy')" :value="proxy?.name" />
+        <UiLabelValue :label="t('snapshot-mode')">
           <template v-if="backupJob.mode" #value>
             <UiTag variant="secondary" accent="info">{{ t(snapshotModeTranslation) }}</UiTag>
           </template>
-        </VtsQuickInfoRow>
-        <VtsQuickInfoRow
+        </UiLabelValue>
+        <UiLabelValue
           :label="t('vm-backup-failure-number-of-retries')"
           :value="
             settings.nRetriesVmBackupFailures !== undefined ? String(settings.nRetriesVmBackupFailures) : undefined
           "
         />
-        <VtsQuickInfoRow :label="t('timeout')" :value="formattedTimeout" />
+        <UiLabelValue :label="t('timeout')" :value="formattedTimeout" />
       </VtsColumn>
       <VtsColumn>
-        <VtsQuickInfoRow
+        <UiLabelValue
           :label="t('speed-limit')"
           :value="maxExportRate ? `${maxExportRate.value} ${maxExportRate.prefix}` : undefined"
         />
-        <VtsQuickInfoRow :label="t('report-when')" :value="reportWhenValueTranslation" />
-        <VtsQuickInfoRow :label="t('report-recipients')">
-          <template #value>
-            <UiTagsList v-if="settings.reportRecipients">
-              <UiTag
-                v-for="(recipient, index) in settings.reportRecipients"
-                :key="index"
-                variant="secondary"
-                accent="info"
-              >
-                {{ recipient }}
-              </UiTag>
-            </UiTagsList>
-          </template>
-        </VtsQuickInfoRow>
-        <VtsQuickInfoRow
+        <UiLabelValue :label="t('report-when')" :value="reportWhenValueTranslation" />
+        <UiLabelValue :label="t('report-recipients')" :value="settings.reportRecipients" />
+        <UiLabelValue
           :label="t('concurrency')"
           :value="settings.concurrency !== undefined ? String(settings.concurrency) : undefined"
         />
       </VtsColumn>
       <VtsColumn>
-        <VtsQuickInfoRow :label="t('compression')" :value="compression" />
-        <VtsQuickInfoRow :label="t('offline-backup')">
+        <UiLabelValue :label="t('compression')" :value="compression" />
```

Comment:
```suggestion
              <UiLabelValue :label="t('compression')">
                  <template #value>
                      <VtsEnabledState :enabled="!!settings.compression" />
                  </template>
             </UiLabelValue>
```

When I see Figma Compression looks like **"offline backup"** or "**shorter backup"**, so why not use the value slot like them?